databases/sqlitebrowser/Portfile

```diff
@@ -6,8 +6,8 @@ PortGroup           cmake 1.1
 PortGroup           github 1.0
 PortGroup           app 1.0
 
-github.setup        sqlitebrowser sqlitebrowser 3.9.1 v
-revision            1
+github.setup        sqlitebrowser sqlitebrowser 3.11.2 v
+revision            0
```

Comment:
we don't usually keep 0 revisions

Reply:
There are 1,677 ports with a 0 revision, but if you feel strongly about it I will remove it.

Reply:
Indeed, yes we do now.